polymesh-docs/polymesh-private/development-environment/walkthrough-rest-api.md

```diff
@@ -715,259 +716,87 @@ Example response:
-              "assetId": "0x35f2d90772b65a565351124987913aea"
-            }
-          ],
-          "accountId": null,
-          "identityId": "0xbf84839e6bc8ad6fe969f560895663fdd2c175b40c9ab3ca63df9fd7010b64f8",
-          "party": "Sender",
-          "legId": 0
-        }
-      ]
-    }
-  }
-}
-```
-
-#### 5.6 Verify or decrypt sender proof using their confidential account to view the amount being sent
-
-```bash
-export RESPONSE_5_6=$(curl --silent --request 'POST' \
-  ''"$PP_REST_API"'/confidential-accounts/'"$RECEIVER_CONFIDENTIAL_ACCOUNT"'/receiver-verify' \
+export RESPONSE_5_5=$(curl --silent --request 'POST' \
+  ''"$PP_REST_API"'/confidential-transactions/'"$CONFIDENTIAL_TRANSACTION_ID"'/auditor-verify' \
   -H 'accept: application/json' \
   -H 'Content-Type: application/json' \
   -d '{
-  "senderProof": "'"$SENDER_PROOF_5_5"'"
+  "auditorKey": "'"$RECEIVER_CONFIDENTIAL_ACCOUNT"'"
 }')
 
-echo $RESPONSE_5_6 | jq
+echo $RESPONSE_5_5 | jq
 ```
 
 Example response:
 
 ```json
 {
-  "isValid": true,
-  "amount": "1000",
-  "errMsg": null
+  "verifications": [
+    {
+      "isAuditor": false,
+      "isProved": true,
+      "assetId": "35f2d907-72b6-5a56-5351-124987913aea",
+      "legId": "0",
+      "isValid": null,
+      "errMsg": null
+    }
+  ]
 }
```

Comment:
There is no amount shown in this example response? The example response in the swagger docs for an auditor verification is as follows and does include an amount:
```
[
  {
    "legId": "1",
    "assetId": "76702175-d8cb-e3a5-5a19-734433351e25",
    "isProved": true,
    "isAuditor": true,
    "amount": "100",
    "isValid": "string",
    "errMsg": "string"
  }
]
```
If this was the actual response it may be that the REST API doesn't properly handle a non auditor confidential account being provided. the response indicated the account was not an auditor but still suggested the ZKP was proven but didn't decrypt an amount.